src/tribol/interface/tribol.hpp

```diff
@@ -218,6 +275,14 @@ void registerNodalVelocities( integer meshId,
                               const real* vy,
                               const real* vz=nullptr );
 
+/*!
+ * @brief Registers a velocity field on a MFEM volume mesh.
+ * 
+ * @param [in] cs_id The ID of the coupling scheme with the MFEM mesh.
+ * @param [in] v MFEM velocity ParGridFunction defined over the volume mesh.
```

Comment:
Add a 'pre' that a call to 'registerMfemMesh()' (or whatever we name it), must be called prior to this routine and that the grid function must be defined over the same mesh that was registered using cs_id (or something like that).

Reply:
Added.